### PR TITLE
Remove Ubuntu 16.04 and 18.04 from Sumaform support

### PR DESCRIPTION
**Pull request by srbarrios**
## What does this PR change?

Don't merge until 31st August
Part of https://github.com/SUSE/spacewalk/issues/21047

NOTE:
It also remove some left-overs for Ubuntu 16.04

### CODE REVIEW

**Review by vandabarata — APPROVED** [review]

File: .travis.yml
Comment (nodeg) [inline .travis.yml:1]: This should either be in a separate commit or be excluded from this PR. It is not related to the removal of Ubuntu 18.04

**Review by nodeg — APPROVED** [review]
LGTM, although I would prefer to either
- edit the commit message/PR description or
- split Ubuntu 16.04 and 18.04 in 2 distinct commits
